compiler-no-html/package.js

```diff
@@ -7,7 +7,7 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: 'svelte-compiler',
-  use: ['svelte:core@1.2.1'],
+  use: ['svelte:core@1.23.4_1'],
```

Comment:
This update shouldn't be necessary because the package is already deprecated at its current version (1.3.0).